mmdeploy/codebase/mmdet/models/dense_heads/gfl_head.py

```diff
@@ -0,0 +1,203 @@
+import torch
```

Comment:
could add copyright at top line
```
# Copyright (c) OpenMMLab. All rights reserved.
```

Reply:
Hi, do I need to resubmit to add copyright:`# Copyright (c) OpenMMLab. All rights reserved.`?
Also, I have a little doubt about the speed test. I have performed the speed test of the model under tensorrt on cuda and cpu respectively. Why is the FPS based on cpu very high, much higher than the speed based on cuda? Only the `--device cpu/cuda` parameter was changed during testing. My laptop cpu is r9-5900hs and gpu is 3060.**But I found that `nvidia-smi` shows gpu usage in both cases.**    
cpu:  `python tools/test.py configs/mmdet/detection/detection_tensorrt_dynamic-320x320-1344x1344.py ../MMDetection/configs/gfl/gfl_r50_fpn_1x_coco.py --model test_work/test_GFL/end2end.engine --speed-test --device cpu`  
![Screenshot_select-area_20220210191630](https://user-images.githubusercontent.com/55585674/153396490-c4c944c2-37b1-40d5-968f-0f0105e0300a.png)  
cuda:  `python tools/test.py configs/mmdet/detection/detection_tensorrt_dynamic-320x320-1344x1344.py ../MMDetection/configs/gfl/gfl_r50_fpn_1x_coco.py --model test_work/test_GFL/end2end.engine --speed-test --device cuda`  
![Screenshot_select-area_20220210191906](https://user-images.githubusercontent.com/55585674/153396751-033f54bd-9459-4cd9-aefc-56b77abc1b1c.png)

Reply:
@RunningLeon

Reply:
> Hi, do I need to resubmit to add copyright:`# Copyright (c) OpenMMLab. All rights reserved.`? Also, I have a little doubt about the speed test. I have performed the speed test of the model under tensorrt on cuda and cpu respectively. Why is the FPS based on cpu very high, much higher than the speed based on cuda? Only the `--device cpu/cuda` parameter was changed during testing. My laptop cpu is r9-5900hs and gpu is 3060.**But I found that `nvidia-smi` shows gpu usage in both cases.** cpu: `python tools/test.py configs/mmdet/detection/detection_tensorrt_dynamic-320x320-1344x1344.py ../MMDetection/configs/gfl/gfl_r50_fpn_1x_coco.py --model test_work/test_GFL/end2end.engine --speed-test --device cpu` ![Screenshot_select-area_20220210191630](https://user-images.githubusercontent.com/55585674/153396490-c4c944c2-37b1-40d5-968f-0f0105e0300a.png) cuda: `python tools/test.py configs/mmdet/detection/detection_tensorrt_dynamic-320x320-1344x1344.py ../MMDetection/configs/gfl/gfl_r50_fpn_1x_coco.py --model test_work/test_GFL/end2end.engine --speed-test --device cuda` ![Screenshot_select-area_20220210191906](https://user-images.githubusercontent.com/55585674/153396751-033f54bd-9459-4cd9-aefc-56b77abc1b1c.png)

@Richard-mei Please add copy right in your commit. BTW, TensorRT is for cuda. Passing `cpu` to `--device` should raise error. @VVsssssk Could check if there is a bug here?